### DISCUSSION TITLE
Possible to add a "Prefer small file size" feature?

### DISCUSSION THREAD

**Discussion by walterhpdx** [pymedusa/Medusa#9533] [Ideas & Feature requests]
The only big thing that I've ever had an issue with is file sizes when downloading new shows that have been on the air for a while.  For example, I recently watched MasterChef series 10, and my husband said to get series 3.  I did, and said "Prefer 480/720, but allow 480/720/1080".  The system was able to get 720 files - however, each episode was between 3Gb and 6Gb.  I honestly don't need to see every one of Gordon Ramsay's pores, and when I did manual searches, there were tons of 400Mb to 750Mb files available, but pyMedusa grabbed the largest files possible.  This is the only reason I don't let pyMedusa put files directly on my Plex server; probably 75% of the time I convert the files with Handbrake to decrease the file size.  I'd love to mostly be able to skip this step and have pyMedusa just slot new episodes into place on the server.

**Comment by p0psicles** [comment]
This isn't going to be added any time soon. But you could check if the releases with a smaller file size are from a specific release group. And use required words
